apps/site/pages/docs/building-sections/overrides/sending-components-to-the-headless-cms.mdx

```diff
@@ -0,0 +1,34 @@
+
+# Sending components to the Headless CMS
```

Comment:
Can we change this title to `Syncing Components with the Headless CMS`. It sounds more natural 🤔